rmgpy/rmg/model.py

```diff
@@ -674,7 +675,15 @@ def enlarge(self, newObject=None, reactEdge=False,
 
             rxns = reactAll(self.core.species, numOldCoreSpecies,
                             unimolecularReact, bimolecularReact, trimolecularReact=trimolecularReact)
-            spcs = [self.retrieveNewSpecies(rxn) for rxn in rxns]
+            #spcs = [self.retrieveNewSpecies(rxn) for rxn in rxns]
+
+            # get new species and save in spcs
+            spcs = []
+            for rxn in rxns:
+                spcs.extend(rxn.reactants)
+                spcs.extend(rxn.products)
+
+            ensure_independent_atom_ids(spcs, resonance=True) 
```

Comment:
Why do you need to ensure_independent_atom_ids here?

Reply:
Otherwise Species.getResonanceHybrid() fails due to not having valid atom ids. We call ensure_independent_atom_ids in generate_reactions_from_families, however, this is not working for multiprocessing. Using ensure_independent_atom_ids here fixes that issue.